app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,54 @@
+class PagesController < ApplicationController
```

Comment:
I think each block/line of code spacing is off by one. Looks like the fix would be to delete a space for each line